# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/streams/ReadableStreamBYOBReader.cpp
# Commit: 4ec8f7211b07
# Full Hash: 4ec8f7211b0705481e717e365724a2e16d8601bb
# Author: Tom Schuster <evilpies@gmail.com>
# Date: 2022-03-19 09:41:58
# Regressor Bug: 1750298
# File Overlap Count: 1
# Description:
#   Bug 1750298 - Use custom ReadRequest for FetchStreamReader. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137681
# ==============================================================================

diff -r 1820f90df625 -r 4ec8f7211b07 dom/streams/ReadableStreamBYOBReader.cpp
--- a/dom/streams/ReadableStreamBYOBReader.cpp	Fri Mar 18 17:23:13 2022 +0000
+++ b/dom/streams/ReadableStreamBYOBReader.cpp	Fri Mar 18 17:46:32 2022 +0000
@@ -125,7 +125,7 @@
     if (aChunk.isObject()) {
       // We need to wrap this as the chunk could have come from
       // another compartment.
-      JS::RootedObject chunk(aCx, &aChunk.toObject());
+      JS::Rooted<JSObject*> chunk(aCx, &aChunk.toObject());
       if (!JS_WrapObject(aCx, &chunk)) {
         return;
       }